vectorlink-worker/src/main.rs

```diff
@@ -29,7 +64,11 @@ async fn main() -> Result<(), Box<dyn std::error::Error>> {
         args.identity.unwrap_or_else(generate_identity),
     )
     .await?;
+    successful_connection_counter.inc();
+
     VectorlinkTaskHandler::process_queue(&mut queue).await?;
+    successful_task_counter.inc();
```

Comment:
We will actually never get here. process_queue will just keep processing tasks indefinitely. that's why there's an `unreachable!()` down there - we never actually expect to get there.